src/Protocol/Protocol_1_14.cpp

```diff
@@ -732,7 +719,9 @@ UInt8 cProtocol_1_14::GetProtocolEntityType(const cEntity & a_Entity) const
 				case PType::pkSplashPotion: return 81;
 				case PType::pkFirework: return 26;
 				case PType::pkWitherSkull: return 92;
+				default: break;
```

Comment:
Not sure about this change, this switch should be an all-cases-handled one so we get a warning when updating the PTypes. (The break on the line below is fine though).

Reply:
Oh yeah - I remember. That's why there are that many useless case statements in the monster section.